cloud/pulsar-admin/README.md

```diff
@@ -12,6 +12,8 @@ The `pulsar-admin` is a CLI tool written in Java language for the Apache Pulsar
 
 # Usage
 
+## Token
```

Comment:
Please help change all the "Token/Oauth2" into  "Token/Oauth2 authentication plugin" in these 2 files?